admin/src/pages/CreationConfirm.vue

```diff
@@ -1,18 +1,11 @@
 <template>
   <div class="creation-confirm">
-    <div v-show="overlay" id="overlay" class="">
-      <b-jumbotron class="bg-light p-4">
-        <template #header>{{ $t('overlay.confirm.title') }}</template>
-        <template #lead>{{ $t('overlay.confirm.text') }}</template>
-        <hr class="my-4" />
-        <p>{{ $t('overlay.confirm.question') }}</p>
-        <b-button size="md" variant="danger" class="m-3" @click="overlay = false">
-          {{ $t('overlay.confirm.no') }}
-        </b-button>
-        <b-button size="md" variant="success" class="m-3 text-right" @click="confirmCreation">
-          {{ $t('overlay.confirm.yes') }}
-        </b-button>
-      </b-jumbotron>
+    <div v-show="overlay" id="overlay" @dblclick="overlay = false">
```

Comment:
```suggestion
    <div v-if="overlay" id="overlay" @dblclick="overlay = false">
```
Hier kommt der Fehler her. Wenn kein item ausgewählt ist, dann ist das Datum natürlich `null` und dan geht `$d` nicht.